src/functions-and-arrays.js

```diff
@@ -47,8 +155,26 @@ const wordsCount = [
   'matter'
 ];
 
+function howManyTimes(arrayGiven, wordGiven) {
+  if (arrayGiven.length === 0) {
+    return 0;
+  }
+  let lasPosition = 0;
+  let counter = 0;
+  //contar cuantas veces aparece una palabra dada recorriendo la array y emprzando cada iteración desde la última posición + 1 donde se encontró la palabra.
+  for (let i = 0; i < arrayGiven.length; i++) {
```

Comment:
No crees que con chequear si la wordGiven es igual a arrayGiven[i] y hacer counter++ no es suficiente?

Reply:
hago el update de la lasPosition porque sino no sé desde que posicioón el indexOf. sino pusiera nada volveria desde el principio y encontrai seiempre la misma posición, no?

Reply:
que pasa si haces asi:

`for(let i = 0; i < arrayGiven.length; i++ ){
  const word = arrayGiven[I];
 if(word == wordGiven) counter++
}
 return counter
`